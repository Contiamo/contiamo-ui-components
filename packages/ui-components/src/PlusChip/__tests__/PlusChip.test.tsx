import * as React from "react"
import { render } from "enzyme"

import ThemelessPlusChip from "../PlusChip"
import wrapDefaultTheme from "../../../utils/wrap-default-theme"

const PlusChip = wrapDefaultTheme(ThemelessPlusChip)

describe("Plus Chip", () => {
  it("Should correctly render", () => {
    expect(render(<PlusChip />)).toMatchSnapshot()
  })
})
