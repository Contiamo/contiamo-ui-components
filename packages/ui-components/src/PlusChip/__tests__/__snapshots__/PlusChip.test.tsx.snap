// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Plus Chip Should correctly render 1`] = `
<div
  className="undefined PlusChip"
  role="button"
  tabIndex={-1}
>
  +
</div>
`;
